src/deb/control/CHANGES.txt

```diff
@@ -0,0 +1,5 @@
+[[name]] ([[version]]) stable; urgency=low
+
+  * See http://jmxtrans.googlecode.com/ for more details.
+
+ -- Pushkar Raste <praste@bloomnberg.net, pushkar.raste@gmail.com> [[buildTimeStamp]]
```

Comment:
That being said, we should ask @lookfirst for his opinion on that one. I think it make sense to add you at least to the list of developers in the `pom.xml`, but again, that's a question for @lookfirst.